test/unit_convex_hull.jl

```diff
@@ -33,6 +33,20 @@ for N in [Float64, Rational{Int}]
     convex_hull!(points_2D) # check in-place version
     @test points_2D == [[N(-1), N(-1)], [N(1), N(0)], [N(1), N(1)], [N(0), N(1)]]
 
+    # three vertex case in 2 dimension
+    ccw_points = [[N(1), N(1)], [N(-1), N(1)], [N(-1), N(0)]]
```

Comment:
Simpler (all over this file):
```suggestion
    ccw_points = [N[1, 1], N[-1, 1], N[-1, 0]]
```